### PR TITLE
Require a server URL

### PR DESCRIPTION
**Pull request by drew-gross**
Throw at launch if no server url is provided. Hopefully this will reduce the number of issues that are opened.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -82,7 +82,7 @@ function ParseServer({
   facebookAppIds = [],
   enableAnonymousUsers = true,
   oauth = {},
-  serverURL = '',
+  serverURL = (()=>{throw 'You must provide a serverURL!'})(),
```
Comment (flovilmart) [inline src/index.js]: let's use the same pattern on appId and masterKey then no?

Comment (drew-gross) [inline src/index.js]: Sure, I don't see why not. I might actually give this pattern a name to make it more clear.

Comment (flovilmart) [inline src/index.js]: Too bad we can't use '!!' as a function name. `notOptional("serverURL")`

**Comment by nlutsenko** [comment]
Squash and merge? Love the 2 liners :)

**Pull request merged**